sonar-python-plugin/src/test/java/org/sonar/plugins/python/indexer/SonarQubePythonIndexerTest.java

```diff
@@ -351,11 +351,11 @@ void test_test_files_not_using_cache() throws IOException, NoSuchAlgorithmExcept
     pythonIndexer = new SonarQubePythonIndexer(inputFiles, cacheContext, context);
     pythonIndexer.buildOnce(context);
 
-    assertThat(pythonIndexer.canBePartiallyScannedWithoutParsing(file1)).isTrue();
+    assertThat(pythonIndexer.canBePartiallyScannedWithoutParsing(file1)).isFalse();
```

Comment:
I am not sure I understand this part. So now that we gather everything together, test and main code, the test files are already cached?